### PR TITLE
saved metatag data for news node

### PR DESCRIPTION
**Pull request by boznik**
# NOT READY FOR REVIEW

# Summary
- Adding metatag data for news to profile

# Needed By (Date)
- Before https://github.com/SU-SOE/soe_profile/pull/12 gets merged

# Urgency
- we need it

# Steps to Test

1. verify metatag setting match basic page

# Affected Projects or Products
- lellandd8 stack

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/D8CORE-1896

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by sherakama** [comment]
@boznik 

You're missing the mark here. Please not only enable the settings so that they match the basic page type but also provide metatag configuration using your best judgment for the following stuff.

* News Node
* Taxonomy Term Page
* Page Manager Page

Right now you have enabled the news content type with exactly the basic page's configuration and this is not good enough. As requested verbally, please update the configuration so that the metatag default data uses the correct fields and has proper OG data. Please create new configuration specifically for the news node so that it doesn't use page content type fields for images. Perhaps add the topics field to the content type as keywords. Please review the options provided by the metatag module and populate the fields that in your opinion are useful or necessary.

**Comment by boznik** [comment]
Ah ok, i thought i was missing something and wondered why you wanted it to match basic page. will modify...

**Comment by boznik** [comment]
I don't see a way to add metatags to page manager pages. i found an old thread about it: https://www.drupal.org/project/page_manager/issues/2711051 .

**Comment by sherakama** [comment]
> I don't see a way to add metatags to page manager pages. i found an old thread about it: https://www.drupal.org/project/page_manager/issues/2711051 .

That issue links off to a new issue (https://www.drupal.org/project/metatag/issues/2563657) which shows a commit to the 8.x-1.x branch of metatag. If you look at the available modules we already have in our LelandD8 build you will also see that we have a `metatag_page_manager` module available. Have you tried that module? If you did and it failed to provide the functionality we are looking for could you try the `metatag_views` module? 

Pending those two options falling over that thread also mentions another contrib module that might be an option: https://www.drupal.org/project/metatag_routes

**Comment by boznik** [comment]
ok, metatag_page_manager was successful.

File: soe_profile.info.yml [hunk 1]
```diff
@@ -102,6 +102,9 @@ dependencies:
   - menu_block:menu_block
   - menu_link_weight:menu_link_weight
   - metatag:metatag
+  - metatag:metatag_open_graph
+  - metatag:metatag_page_manager
+  - metatag:metatag_verification
```
Comment (sherakama) [inline soe_profile.info.yml:107]: I added these as we have been sloppy keeping up with what is in core.extension.

File: config/sync/metatag.metatag_defaults.node.yml [hunk 2]
```diff
@@ -7,8 +7,9 @@ _core:
...
 label: Content
 tags:
+  keywords: '[node:su_news_topics]'
   title: '[node:title] | [site:name]'
   canonical_url: '[node:url]'
   description: '[node:summary]'
-  og_image: '[node:su_page_banner:entity:su_banner_image:0:entity:field_media_image:large]'
-  og_image_url: '[node:su_page_banner:entity:su_banner_image:0:entity:field_media_image:large]'
+  og_image: '[node:su_page_banner:entity:su_banner_image:0:entity:field_media_image:large:node:su_news_featured_media]'
+  og_image_url: '[node:su_page_banner:entity:su_banner_image:0:entity:field_media_image:large:node:su_news_featured_media]'
```
Comment (sherakama) [inline config/sync/metatag.metatag_defaults.node.yml:15]: Ah whoops. These are on the default node setting. We need a separate setting for news items.

**Pull request merged**